packages/syft/src/syft/service/action/action_graph.py

```diff
@@ -148,7 +149,7 @@ def get(self, uid: Any) -> Any:
     def delete(self, uid: Any) -> None:
         raise NotImplementedError
 
-    def find_neighbors(self, uid: Any) -> List[Any]:
+    def find_neighbors(self, uid: Any) -> Optional[List[Any]]:
```

Comment:
should this be
```suggestion
    def find_neighbors(self, uid: UID) -> Optional[List]:
```
? I saw another function with this signature on line 270.

also slight nitpick: could just be `Optional[List]` instead of `Optional[List[Any]]`.